### PR TITLE
Rename "echo-example" to "host-example"

### PR DESCRIPTION
**Pull request by lynn**
"echo-example" and its README sound like it does nothing but echo back datachannel messages.

In reality, it isn't an echo server (it's a [`rev`](https://ss64.com/bash/rev.html) server if anything 🙂), and it _is_ a full-fledged example of a minimalistic Rainway host app.

I also made it print the Peer ID in green, to make the demo experience a little bit smoother.

I moved `service-example` to `host-service-example` to try to convey that they do the same thing, but one is also runnable as a service.

### CODE REVIEW

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -18,27 +18,27 @@ jobs:
...
     # Locally we debug the build output
     - name: List bin (local only)
       if: ${{ env.ACT }}
       run: ls -alR bin
-    
+
     - name: Parse git commit
-      run: echo GH_REL_TAG=$(git rev-parse --short HEAD 2> /dev/null | sed "s/\(.*\)/v0-\1/") >> $GITHUB_ENV
-    
+      run: host GH_REL_TAG=$(git rev-parse --short HEAD 2> /dev/null | sed "s/\(.*\)/v0-\1/") >> $GITHUB_ENV
```
Comment (bengreenier) [inline .github/workflows/cd.yml:36]: not this 😸

File: .github/workflows/cd.yml [hunk 2]
```diff
@@ -18,27 +18,27 @@ jobs:
...
+
     # Locally we debug the tag parse
-    - run: echo ${{ env.GH_REL_TAG }}
+    - run: host ${{ env.GH_REL_TAG }}
```
Comment (bengreenier) [inline .github/workflows/cd.yml:39]: not this 😸

File: .github/workflows/cd.yml [hunk 3]
```diff
@@ -47,10 +47,10 @@ jobs:
     - name: Package bin
       run: |
         mkdir rel
-        zip rel/echo-example-${{ env.GH_REL_TAG }}.zip -r bin/echo-example &&
-        zip rel/service-example-${{ env.GH_REL_TAG }}.zip -r bin/service-example &&
-        echo "Packaged."
-    
+        zip rel/host-example-${{ env.GH_REL_TAG }}.zip -r bin/host-example &&
+        zip rel/host-service-example-${{ env.GH_REL_TAG }}.zip -r bin/host-service-example &&
+        host "Packaged."
```
Comment (bengreenier) [inline .github/workflows/cd.yml:52]: not this 😸

File: host-example/Program.cs [hunk 4]
```diff
@@ -38,20 +30,26 @@ static string ReverseString(byte[] data)
...
+        Array.Reverse(chars);
+        peer.Send(channel, new string(chars));
+    },
 };
 
 // initalize the runtime
 using var runtime = await RainwayRuntime.Initialize(config);
 Console.WriteLine($"Rainway SDK Version: {runtime.Version}");
-Console.WriteLine($"Peer Id: {runtime.PeerId}");
+Console.ForegroundColor = ConsoleColor.Green;
```
Comment (bengreenier) [inline host-example/Program.cs:50]: nice 🚀

**Review by bengreenier — CHANGES_REQUESTED** [review]
Good work @lynn ! A few CI things that should not be changed, otherwise LGTM! Let's also make sure we update https://docs.rainway.com/docs/what-is-rainway#quick-demo when this lands

File: .github/workflows/cd.yml (hunk 1 above)
Comment (lynn) [reply .github/workflows/cd.yml:36]: LOL

File: .github/workflows/cd.yml (hunk 3 above)
Comment (bengreenier) [reply .github/workflows/cd.yml:52]:
```suggestion
        echo "Packaged."
```

File: .github/workflows/cd.yml (hunk 2 above)
Comment (bengreenier) [reply .github/workflows/cd.yml:39]:
```suggestion
    - run: echo ${{ env.GH_REL_TAG }}
```

File: .github/workflows/cd.yml (hunk 1 above)
Comment (bengreenier) [reply .github/workflows/cd.yml:36]:
```suggestion
      run: echo GH_REL_TAG=$(git rev-parse --short HEAD 2> /dev/null | sed "s/\(.*\)/v0-\1/") >> $GITHUB_ENV
```

**Pull request merged**